driver/tke_driver.go

```diff
@@ -564,8 +603,83 @@ func getState(info *types.ClusterInfo) (*state, error) {
 
 // Update implements driver update interface
 func (d *Driver) Update(ctx context.Context, info *types.ClusterInfo, opts *types.DriverOptions) (*types.ClusterInfo, error) {
-	logrus.Info("unimplemented")
-	return nil, fmt.Errorf("not implemented")
+	logrus.Info("Invoking update cluster")
+	state, err := getState(info)
+	if err != nil {
+		return nil, err
+	}
+
+	newState, err := getStateFromOpts(opts, false)
+	if err != nil {
+		return nil, err
+	}
+
+	svc, err := getTKEServiceClient(state, "POST")
+	if err != nil {
+		return nil, err
+	}
+
+	logrus.Debugf("Updating config, clusterName: %s, clusterVersion: %s, new node: %v", state.ClusterName, state.ClusterVersion, newState.GoodsNum)
+
+	if newState.GoodsNum != 0 {
```

Comment:
should we compare new node count with the current node count also? I don't think we need to send update request if the node count is not changed.